Resources/Prototypes/Entities/Objects/Specific/chemistry-bottles.yml

```
@@ -14,7 +14,7 @@
       Glass: 25
   - type: SpaceGarbage
   - type: Sprite
-    sprite: Objects/Specific/Chemistry/bottle.rsi
+    sprite: _DV/Objects/Specific/Chemistry/bottle.rsi #delta-V Change - resprited some med items
```

Missed this on my first pass. The Delta-V comments (across all non-`_DV` files, only leaving the comment here for demonstration) should be `# Delta-V - ...`
Recommend find-and-replace `#delta-V Change` with `# Delta-V -`, and I think ideally copypasting the resprite comment to every comment you made here
```suggestion
    sprite: _DV/Objects/Specific/Chemistry/bottle.rsi # Delta-V - resprited some med items
```

---

Ill see if i can get that change made!

---

may be a moment.

---

Is the hyhpen between Delta and V standard?

---

> Is the hyhpen between Delta and V standard?
Yes it is

---

wrong its # DeltaV for comments read CONTRIBUTING.md